cmake/macros/macro_expand_instantiations.cmake

```diff
@@ -78,7 +78,12 @@ MACRO(EXPAND_INSTANTIATIONS _target _inst_in_files)
   #
   FOREACH(_build ${DEAL_II_BUILD_TYPES})
     STRING(TOLOWER ${_build} _build_lowercase)
+
     ADD_DEPENDENCIES(${_target}.${_build_lowercase} ${_target}.inst)
```

Comment:
I mean, we do describe the generator rule for the `.inst` file and if we set `OUTPUTS` on it correctly, shouldn't it run this rule when the output is out of date and/or is needed?